continuous-integration/linux/lint.sh

```diff
@@ -16,7 +16,7 @@ echo "Checking formatting"
 errorsFound=0
 for fileName in $cppFiles $hFiles; do
     echo $fileName
-    diff $fileName <(clang-format-8 $fileName)
+    diff $fileName <(clang-format $fileName)
```

Comment:
You will get hard to understand errors if clang-format is not version 10, better to just get an error about clang-format-10 not exisiting.

Reply:
done